.gitignore

```diff
@@ -15,6 +15,7 @@
 **/dist
 **/node_modules
 .idea/*
+.vscode
 
```

Comment:
add .DS_Store to gitignore

Reply:
I have had .DS_Store show up sometimes in the project (and I would manually delete it), but sometimes it doesn't show up. What causes this to be generated?

Reply:
Macbook automatically generates those files, idk the reason never really researched it